src/scraper/scrapers/base_scraper.py

```diff
@@ -0,0 +1,60 @@
+"""
+    Base class for a scrapper.
+    In order to create and wire a new scrapper:
+        1) Create a new scraper in the "scrapers" directory
+        2) Make your scraper a subclass of BaseScraper
+        3) Implement missing methods (parse & scrape)
+        4) add an entry in settings.py to the URL_TO_SCRAPER map, maping from 
+           a domain name to your new scraper. Import it if necessary
+"""
+
+# Python imports
+from typing import List
+
+
+class BaseScraper:
+    """
+        Base class for scrapers implementations
+    """
+
+    def parse(self, response) -> dict:
```

Comment:
NIT: I noticed that you are following the typing which is great.
If we would like to be strict, the typing here has two mistakes:
1. response has no type: maybe is because Scrapy does not have type hints. Is that right?
2. dict is not a valid type hint. The `typing` module defined a type called `Dict`. That one is Generic and a typical annotation example is like this:
```
def parse(self, response: MyType) -> Dict[str, Any]:
```

Reply:
About 1.
 No, it does have type hints, we decided to keep it as "any" type so every scraper could have its own response type according to its needs. But I'll agree that we should have a base class in order to have a minimal interface for every response object, and (using scrapy as an example) a ScrapyResponse implementation for scrapy Response object could just delegate the interface implementation to scrapy functions, for example:
 
 ```
class BaseResponse:
   def url(self) -> str: # just an example
      pass
      
class ScrapyResponse(BaseResponse):
   def __init__(self, scrapy_response : scrapy.http.Response):
      self.scrapy_response = scrapy_response
      
   def url(self) -> str:
      return self.scrapy_response.url()
 ```
 I'll be addressing this issue in some other PR
 About 2.
 You're right, I should change every dict signature. I wasn't sure which one I should have used when I first wrote it. I'll change it ASAP